cluster/gce/config-common.sh

```diff
@@ -60,3 +60,8 @@ if [[ "${FEDERATION:-}" == true ]]; then
 else
     NODE_SCOPES="${NODE_SCOPES:-compute-rw,monitoring,logging-write,storage-ro}"
 fi
+
+# Fluentd requirements
+FLUENTD_GCP_MEMORY_LIMIT="300Mi"
```

Comment:
Could you please follow the above pattern with defaults? Sth like (to be verified)
```
FLUENTD_GCP_MEMORY_LIMIT="${FLUENTD_GCP_MEMORY_LIMIT:-"300Mi"}"
```